### ISSUE TITLE
Large File Size causing issue with static analysis

### ISSUE THREAD

**Issue by ajinabraham** [MobSF/Mobile-Security-Framework-MobSF#1679]
## ENVIRONMENT

```
OS and Version:  Ubuntu Focal
Python Version:  3.8
MobSF Version:  3.3.1
```

## EXPLANATION OF THE ISSUE

See: https://mobsf.slack.com/archives/C0QMLNKMF/p1613679967136900?thread_ts=1598238085.015300&cid=C0QMLNKMF

## STEPS TO REPRODUCE THE ISSUE

Reproduced in PWD.
APK attached to slack thread.

## LOG FILE

```
Paste the contents of mobsf/debug.log here or attach the log file.
```

**Comment by ajinabraham** [comment]
After inspection, this looks like a very large opengl file  d5f52c57fea09279921f014195b48ac7/java_source/org/lwjgl/opengl/ContextCapabilities.java has around `86464338` bytes of data and has around 1,74,000 + lines of code in a single file. We will probably add logic in libsast to ignore files greater than a specific size.

**Comment by ajinabraham** [comment]
Bump libsast to 1.3.9 to fix this, also MobSF 3.3.2 will be using this version of libsast
